embedding_matrix_api/app/api.py

```diff
@@ -0,0 +1,85 @@
+"""API module."""
+
+import logging
+
+from litestar import Router, get, post
+from litestar.enums import RequestEncodingType
+from litestar.params import Body
+from litestar.status_codes import (
+    HTTP_200_OK,
+)
+
+from model import Model
+from schemas import (
+    HealthResponse,
+    ModelInputData,
+    PredictResponse,
+)
+from settings import (
+    api_settings,
+    model_settings,
+    transformer_settings,
+)
+
+model = Model(
```

Comment:
@vicente87 Why do you use the `Model` into de embedding matrix service?

Reply:
Here we do not use it but as I told you before that we had taken the pattern of services that you gave us and we did not modify many things and that is why they may not be used